hack/jenkins/upload-to-gcs.sh

```diff
@@ -157,22 +157,20 @@ function update_job_result_cache() {
   for upload_attempt in $(seq 3); do
     echo "Copying ${job_results} to ${tmp_results} (attempt ${upload_attempt})"
     gsutil -q cat ${job_results} 2>&- |\
-     sed -n 's/^\({"version".*}\),*/\1,/p' >> ${tmp_results} || continue
-
+     sed -n 's/^\({"version".*}\),*/\1,/p' |\
```

Comment:
Pardon.  No, that's just a ridiculously compressed compound sed expression that does a few things.  Updated with a comment.